fragments/k8s/core/bundle.yaml

```diff
@@ -30,6 +30,7 @@ services:
     options:
       channel: 1.7/stable
     expose: true
+    constraints: "cores=4 mem=4G"
```

Comment:
Me neither! After testing, though, I think we do.

If you don't have constraints on the machine, then the initial deploy will have worker on a 1-core machine.

If you don't have constraints on the application, then `juju get-constraints kubernetes-worker` returns blank. I think that means new units created with `juju add-unit` won't get the constraints, but I didn't confirm that.